mas-foundation/src/main/java/com/ca/mas/core/storage/securestoragesource/MASSecureStorageSource.java

```diff
@@ -0,0 +1,89 @@
+package com.ca.mas.core.storage.securestoragesource;
```

Comment:
Prefer to make the package com.ca.mas.core.storage instead of com.ca.mas.core.storage.securestoragesource.
With everything under com.ca.mas.core.storage, we can hide (package class) classes to the external developer, for example, StorageActions, AccountManagerUtil, ..... the class the more for internal.